### ISSUE TITLE
Setting device to iPhone 5 doesn't appear to work

### ISSUE THREAD

**Issue by kylef** [supermarin/xclisten#16]
Using `-d iphone5` runs xcodebuild with `'name=iPhone Retina (4-inch 64-bit)'`. The iPhone 5 is not 64-bit.

```
Usage: xclisten [optional flags]
        --osx                        Run with OSX sdk (without simulator)
        --ios                        [DEFAULT] Run with iOS sdk
    -d, --device                     Simulated device [iphone5s, iphone5, iphone4, ipad2, ipad4, ipad_air]. Default is iphone5s
    -s, --scheme SCHEME              BYOS (Bring your own scheme)
    -w, --workspace WORKSPACE        BYOW (Bring your own workspace)
    -h, --help                       Show this message
    -v, --version                    Show version
```

``` bash
$ xclisten -d iphone5
Started xclistening to FitnessFirst.xcworkspace
xcodebuild -workspace FitnessFirst.xcworkspace -scheme FitnessFirst -sdk iphonesimulator -destination 'name=iPhone Retina (4-inch 64-bit)' test 2> xcodebuild_error.log | xcpretty -tc
```

**Comment by kylef** [comment]
Same problem with 'iphone4' using iPhone 5S.

**Comment by kylef** [comment]
Also side note, the derived data is saving the iPhone 5S (64-bit) stuff in where Xcode expects the 32-bit is. So any subsequent builds with the other architecture will break.

```
ld: warning: ignoring file /Users/kylef/Library/Developer/Xcode/DerivedData/FitnessFirst-ehlmmtklfpmwrjgydjgjpuufphxg/Build/Products/Debug-iphonesimulator/FitnessFirst.app/FitnessFirst, file was built for x86_64 which is not the architecture being linked (i386): /Users/kylef/Library/Developer/Xcode/DerivedData/FitnessFirst-ehlmmtklfpmwrjgydjgjpuufphxg/Build/Products/Debug-iphonesimulator/FitnessFirst.app/FitnessFirst
ld: warning: ignoring file /Users/kylef/Library/Developer/Xcode/DerivedData/FitnessFirst-ehlmmtklfpmwrjgydjgjpuufphxg/Build/Products/Debug-iphonesimulator/libPods-FitnessFirstTests.a, file was built for archive which is not the architecture being linked (i386): /Users/kylef/Library/Developer/Xcode/DerivedData/FitnessFirst-ehlmmtklfpmwrjgydjgjpuufphxg/Build/Products/Debug-iphonesimulator/libPods-FitnessFirstTests.a
```

**Comment by supermarin** [comment]
@kylef fixed in a251fc2

I'd appreciate if you could open a separate PR and give just a bit more context how it affects the real life usage
